worlds/sc2/locations.py

```diff
@@ -2783,6 +2783,97 @@ def get_locations(world: Optional['SC2World']) -> Tuple[LocationData, ...]:
         make_location_data(SC2Mission.SMASH_AND_GRAB_P.mission_name, "Defeat Kerrigan", SC2_RACESWAP_LOC_ID_OFFSET + 1607, LocationType.MASTERY,
             logic.protoss_competent_comp
         ),
+        make_location_data(SC2Mission.THE_MOEBIUS_FACTOR_Z.mission_name, "Victory", SC2_RACESWAP_LOC_ID_OFFSET + 1900, LocationType.VICTORY,
+            lambda state: (
+                logic.zerg_basic_kerriganless_anti_air(state)

```

Comment:
Can all your AA being a crawler to fight Brood Lords?

Reply:
The Terran version can force the player to beat the mission with Missile Turrets, Medivacs, and Firebats. So... I'd argue yes. The rescues go a long way, and the Zerg version gives you a LOT of Hunter Killers.